catalog-entities/marketplace/plugins/servicenow.yaml

```
@@ -28,7 +28,9 @@ metadata:
       url: https://github.com/backstage/community-plugins/tree/main/workspaces/servicenow
 
 spec:
-  support: dev-preview
+  support:
+    provider: Red Hat
```

```suggestion
    provider: Backstage Community
```

When its not DP, TP @benwilcock , we don't say we support it, or?